### PR TITLE
Updated fontawesome to 4.7

### PR DESCRIPTION
**Pull request by rakesh1988**
Updated fontawesome to 4.7 and made relevant changes to string resource and java file

### CODE REVIEW

File: fontawesome-typeface-library/build.gradle [hunk 1]
```diff
@@ -8,8 +8,8 @@ android {
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
         consumerProguardFiles 'consumer-proguard-rules.pro'
-        versionCode 46003
-        versionName "4.6.0.3"
+        versionCode 47000
+        versionName "4.7"
```
Comment (mikepenz) [inline fontawesome-typeface-library/build.gradle]: please change to 4.7.0.0

File: fontawesome-typeface-library/gradle.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
-VERSION_NAME=4.6.0.3
-VERSION_CODE=46003
+VERSION_NAME=4.7
```
Comment (mikepenz) [inline fontawesome-typeface-library/gradle.properties]: please change to 4.7.0.0

File: fontawesome-typeface-library/src/main/java/com/mikepenz/fontawesome_typeface_library/FontAwesome.java [hunk 3]
```diff
@@ -70,7 +70,7 @@ public String getFontName() {
 
     @Override
     public String getVersion() {
-        return "4.6.0.3";
+        return "4.7";
```
Comment (mikepenz) [inline fontawesome-typeface-library/src/main/java/com/mikepenz/fontawesome_typeface_library/FontAwesome.java]: please change to 4.7.0.0

File: fontawesome-typeface-library/src/main/res/values/library_font_awesome_strings.xml [hunk 4]
```diff
@@ -6,7 +6,7 @@
     <string name="library_FontAwesome_libraryName">Font Awesome</string>
     <string name="library_FontAwesome_libraryDescription">Font Awesome gives you scalable vector icons that can instantly be customized - size, color, drop shadow, and anything that can be done with the power of CSS.</string>
     <string name="library_FontAwesome_libraryWebsite">http://fontawesome.io/icons/</string>
-    <string name="library_FontAwesome_libraryVersion">4.6.0.3</string>
+    <string name="library_FontAwesome_libraryVersion">4.7</string>
```
Comment (mikepenz) [inline fontawesome-typeface-library/src/main/res/values/library_font_awesome_strings.xml]: please change to 4.7.0.0

**Comment by mikepenz** [comment]
Thank you :)

**Comment by rakesh1988** [comment]
Changed 👍

**Comment by mikepenz** [comment]
Awesome. thank you very much.
testing in a few seconds. ;)

**Comment by rakesh1988** [comment]
Thanks to you too for providing such awesome libraries.....

**Comment by mikepenz** [comment]
thanks for helping to make it even better ;)

**Comment by rakesh1988** [comment]
@mikepenz I think I have made a mistake. I have not deleted the old ttf file. So, when users add ```compile 'com.mikepenz:fontawesome-typeface:4.7.0.0@aar'``` it will increase the size by ~148KB. How shall I go ahead? Another Pull or you can delete?

**Comment by mikepenz** [comment]
@rakesh1988 oh no problem. I'll delete it

**Comment by mikepenz** [comment]
update it on bintray and maven cetnral. your local caches though could still have the old version (I did not change the version number)

**Pull request merged**